public/templates/loop.php

```diff
@@ -1,7 +1,7 @@
 <li class="isotope-item <?php echo esc_attr( $term_class ); ?> all">
-<?php if( has_post_thumbnail( $isotope_loop->ID ) ) : ?>
+<?php if( has_post_thumbnail( $this->isotope_loop->ID ) ) : ?>
     <a href="<?php the_permalink(); ?>" title="<?php the_title(); ?>">
-        <?php the_post_thumbnail( $isotope_loop->ID, 'medium' ); ?>
+        <?php the_post_thumbnail( $this->isotope_loop->ID, 'medium' ); ?>
```

Comment:
Can you empty all params for the_post_thumbnail function?
Same as previous comment.